src/components/Centre/CentrePanelMiddle.js

```diff
@@ -2,23 +2,44 @@ import React from 'react';
 import './CentrePanel.css';
 import QuoteForm from '../Forms/QuoteForm';
 import Quote from '../../models/Quote/Quote';
+import { useHistory } from 'react-router-dom';
 
 export default function CentrePanelMiddle(){
 
+    let history = useHistory();
+    
     const quote = Quote();
     const formRecord = quote.getFormRecord();
 
+
     async function handleSubmit(record) {
+
         quote.setFormRecord(record);
-        const response = await quote.submitQuote();
-        console.log('Centre Panel Middle.js', response);
-        //return
-    }
+        try{const response = await quote.submitQuote()
+        console.log('Centre Panel Middle.js', response);     
+        
+        const premiumAmount = response.data.data.quote.amount;
+        
+        if(Number.isInteger(Number(premiumAmount)) > 0){
```

Comment:
try to handle logics in the quote service

Reply:
ideally, it would look like this
```
quoteService.submit().then(() => {
    history.push({pathname: 'quote-result'})
}).catch((err) => {
    notification.show('error', 'Oops, something went wrong')
});

Reply:
Changes updated as per review